praw/models/reddit/live.py

```diff
@@ -345,6 +345,24 @@ def discussions(self, **generator_kwargs):
         url = API_PATH['live_discussions'].format(id=self.id)
         return ListingGenerator(self._reddit, url, **generator_kwargs)
 
+    def report(self, type):  # pylint: disable=redefined-builtin
```

Comment:
I think explicit is better because this function should be used carefully.

> Out of curiosity, what happens if you provide it with an invalid type value?

Sorry, I don't know. It should be test actually but I don't want to get banned because of many test reporting ;)